src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/reset.scss">
+    <link rel="stylesheet" href="styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__img"
+        src="./images/imac.jpeg"
+        alt="APPLE iMac 27 Retina 5K"
+      >
+      <a class="card__desc-link" href="#">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+      <p class="card__desc-text">Product code: 195434</p>
+      <div class="stars">
+        <ul class="stars__list stars__list--4">
+          <li class="stars__item">
+          </li>
+
```

Comment:
Fix it everywhere
```suggestion
          <li class="stars__star"></li>
```

Reply:
![Screenshot_3](https://github.com/mate-academy/layout_product-cards/assets/149514552/6ab1bbee-8de7-4b7c-9e2c-9ab7ffb8b365)